### ISSUE TITLE
Get integration pipeline running

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#12]
## User story
1. As a SD
2. I want / need deploy code to the RPI
3. So that we can get start working

## Acceptance criteria
* Code is deployed

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by cat24max** [comment]
I think we should not do an integration pipeline right now, but rather start working on the Python code for 1-2 weeks. Testing it locally is fine for now.

**Comment by usmanjanjua786** [comment]
Better Idea, that can be done at a later stage once code starts running
